tgui/packages/tgui/layouts/Window.js

```diff
@@ -73,15 +73,10 @@ export class Window extends Component {
 }
 
 const WindowContent = props => {
-  const { scrollable, children } = props;
-  // A bit lazy to actually write styles for it,
-  // so we simply include a Box with margins.
+  const { scrollable, className, children } = props;
   return (
-    <Layout.Content
-      scrollable={scrollable}>
-      <Box m={1}>
-        {children}
-      </Box>
```

Comment:
This currently breaks the crew monitor UI, however it doesnt break anything else.
![image](https://user-images.githubusercontent.com/25063394/93112867-65c04d80-f6b0-11ea-8d64-5524bcec3ba5.png)
![image](https://user-images.githubusercontent.com/25063394/93112881-69ec6b00-f6b0-11ea-9bb9-bc8f9c1b7bf8.png)

Other UIs for context.
![image](https://user-images.githubusercontent.com/25063394/93113254-dc5d4b00-f6b0-11ea-8e66-1cb673d60acf.png)

TLDR: Its not perfect, but its "manageable"

Reply:
Added a class to solve this.